rpm/dovecot-ceph-plugin.spec

```diff
@@ -8,8 +8,33 @@
 # License version 2.1, as published by the Free Software
 # Foundation.  See file COPYING.
 
-%{!?dovecot_devel: %define dovecot_devel dovecot22-devel}
 %{!?librados_version: %define librados_version 10.2.5}
```

Comment:
Minimum RADOS version is 12.2.0

Reply:
Is it really? What is missing in 10.2.5? No objection, I'm fine with the requirement, I would like only to understand it.

Reply:
The plugin is not tested with 10.2.5, all our dev, unit and integration tests are running against luminous 12.2.0. Last time i checked, i also ran into some runtime issues with the storage and dictionary plugin using the 10.2.5 on a suse test server with imaptest. However this may already be fixed because we changed a lot since then. I checked the build against 10.2.5 and it builds fine and tests are working also, so i guess it's possible to use the 10.2.5 as minimum now, but i would recommend to set the minimum to 12.2.0 cause this is the minimum version we are working with and supporting in the future.